docs/standard/parallel-programming/index.md

```diff
@@ -1,12 +1,11 @@
 ---
-title: "Parallel Programming in .NET"
-description: Learn about parallel programming in .NET. Use a .NET runtime, class library types, and diagnostic tools to simplify .NET development.
-ms.date: 09/12/2018
+title: "Parallel Programming in .NET: a guide to the documentation"
+description: A list of articles about parallel programming in .NET.
+ms.date: 03/02/2022
 helpviewer_keywords:
   - "parallel programming"
-ms.assetid: 4d83c690-ad2d-489e-a2e0-b85b898a672d
 ---
-# Parallel Programming in .NET
+# Parallel programming in .NET: a guide to the documentation
```

Comment:
```suggestion
# Parallel programming in .NET: A guide to the documentation
```

See https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33685:

If a title or heading includes a colon, capitalize the first word after it.